lab/src/main/kotlin/materialui/lab/components/treeView/treeView.kt

```diff
@@ -0,0 +1,38 @@
+package materialui.lab.components.treeView
+
+import kotlinx.html.DIV
+import kotlinx.html.Tag
+import kotlinx.html.TagConsumer
+import materialui.components.StandardProps
+import materialui.components.toolbar.enums.ToolbarStyle
+import org.w3c.dom.events.Event
+import react.RBuilder
+import react.RClass
+import react.ReactElement
+
+@JsModule("@material-ui/lab/TreeView")
+private external val treeViewModule: dynamic
+
+external interface TreeViewProps : StandardProps {
+    var defaultCollapseIcon: ReactElement?
+    var defaultEndIcon: ReactElement?
+    var defaultExpandIcon: ReactElement?
+    var defaultParentIcon: ReactElement?
+    var expanded: Array<String>?
+    var defaultExpanded: Array<String>?
+    var onNodeToggle: (event: Event, nodeIds: Array<String>) -> Unit?
```

Comment:
`defaultSelected`/`multiSelect`/`selected`/`onNodeSelect` props is missing

https://material-ui.com/api/tree-view/#props